src/amo/server/base.js

```diff
@@ -193,8 +193,8 @@ function baseServer(
   app.use(middleware.csp());
 
   // Serve assets locally from node ap (no-op by default).
-  if (config.get('enableNodeStatics')) {
-    app.use(middleware.serveAssetsLocally());
+  if (config.get('enableNodeStatics') && config.get('staticPath')) {
```

Comment:
Could you keep the previous condition (without `config.get('staticPath')`)? Otherwise we need more tests to cover the two conditions.

`staticPath` isn't an optional/nullable config param, though, which is why I think we can keep the previous condition.

Reply:
Yep, I was a bit paranoid but that shouldn't be needed.